libyul/SideEffects.h

```diff
@@ -42,17 +56,16 @@ struct SideEffects
 	/// If true, the code can be removed without changing the semantics as long as
 	/// the whole program does not contain the msize instruction.
 	bool sideEffectFreeIfNoMSize = true;
-	/// If false, storage is guaranteed to be unchanged by the code under all
-	/// circumstances.
-	bool invalidatesStorage = false;
-	/// If false, memory is guaranteed to be unchanged by the code under all
-	/// circumstances.
-	bool invalidatesMemory = false;
+	/// If false, the execution can potentially go in an infinite loop.
```

Comment:
Please clarify whether or not this includes opcodes / functions that could do external calls and those calls could go into infinite loops.